Dockerfile

```diff
@@ -74,3 +74,25 @@ ENV NODE_EXTRA_CA_CERTS=/usr/local/share/ca-certificates/extra-ca-certificates.c
 EXPOSE 42069
 
 CMD [ "pnpm", "start" ]
+
+########################################
+# STEP 4: DAPP-CONSOLE-API
+########################################
+
+FROM node:18.19-bullseye-slim AS dapp-console-api
```

Comment:
we'll want this docker image to look more like what the paymaster is doing https://github.com/ethereum-optimism/ecosystem/blob/7c10c041d22e1acf6247ffc3836af2d979fc9191/Dockerfile#L38-L53. we want the base OS to be alpine and not bullseyes, the reason the event-log-indexer is doing it this way is ponder needed a slightly different setup with more deps

Reply:
+1,

also one thing to note is the builder step builds **all** of the apps, so no need to rebuild, or refetch any dependencies

pnpm deploy the app then simply copy it in
```
RUN pnpm deploy --filter=dapp-console-api --prod /dapp-console-api

```

Reply:
thank you, good suggestions!